<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Framelogin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="LabelUser">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelSenha">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextUser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextPasswd">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BotaoAcessar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Acessar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPagina_do_Votante">
    <property name="title">
     <string>Pagina do Votante</string>
    </property>
   </widget>
   <addaction name="menuPagina_do_Votante"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
